pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -1019,8 +1019,21 @@ func PublishDeviceInfoToZedCloud(ctx *zedagentContext) {
 
 	//Operational information about Data Security At Rest
 	ReportDataSecAtRestInfo := new(info.DataSecAtRest)
-	ReportDataSecAtRestInfo.Status, ReportDataSecAtRestInfo.Info =
+	var vaultList []vaultmgr.VaultStatus
+	ReportDataSecAtRestInfo.Status, ReportDataSecAtRestInfo.Info, vaultList =
 		vaultmgr.GetOperInfo()
```

Comment:
Isn't vaultmgr going to run in a separate service domain to get sufficient defense in depth?
If so I think vaultmgr should publish this info so that zedagent can subscribe to it.

Reply:
Addressed in the latest patch.